etl-repair.md

```diff
@@ -0,0 +1,95 @@
+# Repair Kubecost ETLs
+
+> **Note**: Configuring [ETL Backups](./etl-backup.md) can prevent situations where you would need to repair large amounts of missing ETL data.
+
+Kubecost's ETL is a computed cache built upon Prometheus metrics and cloud billing data, from which nearly all API requests made by the user and the Kubecost frontend currently rely upon.
+
+The ETL data is stored in a `PersistentVolume` mounted to the `kubecost-cost-analyzer` pod. In the event that you lose or are looking to rebuild the ETL data, the following endpoints should be used.
```

Comment:
Does PersistentVolume need to be formatted with backticks here? It sounds like we're using the term in a general sense than a reference to specific code.

Reply:
Other than this, great doc

Reply:
`PersistentVolume` is a Kubernetes specific term and is how the Kubernetes project writes it.
https://kubernetes.io/docs/concepts/storage/persistent-volumes/

Reply:
@bstuder99 It doesn't need backticks, but since `PersistentVolume` and `kubecost-cost-analyzer` both reference Kubernetes objects, I thought adding the backtick formatting would add good distinction/clarity to the sentence.